### ISSUE TITLE
When completion inserted it deletes char after itself

### ISSUE THREAD

**Issue by TuM0xA-S** [kak-lsp/kak-lsp#501]
how that happens:
when using gopls:
1)  you should pick function(have parens)
2) and after that pick something without brackets(like variable)
3) press <c-o> and see char after deleted

another way
1) pick function
2) press backspace

demo btw https://asciinema.org/a/BPYkULSeAaXy8mXBdKG6upBuc

looks like yellow stuff deleting

**Comment by krobelus** [comment]
Duplicate of #282

**Comment by krobelus** [comment]
Use `snippet_support = false` in your `kak-lsp.toml` as a workaround. Maybe we should do that by default? At least for problematic servers?

I found your screencast is hard to follow because most colors are dark. Consider using a colorscheme that works better with your terminal colors.

**Comment by TuM0xA-S** [comment]
thx now it works. sorry for colors in recording(problems with true color)
